piv/key.go

```diff
@@ -98,7 +98,7 @@ func (a *Attestation) addExt(e pkix.Extension) error {
 	} else if e.Id.Equal(extIDSerialNumber) {
 		var serial int64
 		if _, err := asn1.Unmarshal(e.Value, &serial); err != nil {
-			return fmt.Errorf("parsing serial number: %v", err)
+			return fmt.Errorf("parsing serial number: %w", err)
```

Comment:
Please only wrap errors that are explicitly need to be passed through. I definitely don't want to support users errors.As'ing to things like asn1 error types :)

This applies to a lot of the s/%v/%w/g changes. Please revert those.

Reply:
Done.   I only kept those that were likely to wrap an `apduErr`.